### PR TITLE
Document chunk animator

### PR DESCRIPTION
**Pull request by RDIL**
## Description  
Documents chunk animator mod

## Context  
Requested by @CoalCoding on #580

### CODE REVIEW

**Comment by RDIL** [comment]
@CoalCoding there’s a dupe because the two were behaving the same

**Comment by RDIL** [comment]
Go away grammar police

**Comment by asbyth** [comment]
well fix it !

File: src/main/java/cc/hyperium/mods/chunkanimator/AnimationHandler.java [hunk 1]
```diff
@@ -72,28 +72,28 @@ public void preRenderChunk(RenderChunk renderChunk) {
...
-                        mode = "0";
+                        mode = "Up from ground";
                     } else {
-                        mode = "1";
+                        mode = "Down from sky";
                     }
                 }
-                if (mode.equals("4")) {
-                    mode = "3";
+                if (mode.equals("From direction your facing")) {
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/mods/chunkanimator/AnimationHandler.java]: Incorrect "your".

File: src/main/java/cc/hyperium/mods/chunkanimator/ChunkAnimatorConfig.java [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
-            "0",
-            "1",
-            "2",
-            "3",
-            "4"
+            "Up from ground",
+            "Down from sky",
+            "Down from sky",
+            "From sides",
+            "From direction your facing"
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/mods/chunkanimator/ChunkAnimatorConfig.java]: Incorrect "your".

**Comment by RDIL** [comment]
Oops sorry

**Comment by RDIL** [comment]
@CoalCoding fixed.

**Comment by asbyth** [comment]
so u do it when he asks but not when i ask ok understandable

**Comment by RDIL** [comment]
Well if getting merged is on the line I’m willing to do whatever I can.

**Comment by RDIL** [comment]
e

**Comment by RDIL** [comment]
@CoalCoding ping since it’s fixed and ready for merge, I set it to the upstream/chunk-anim branch as the destination.

**Pull request merged**